apps/web/src/pages/templates/components/SubPageWrapper.tsx

```diff
@@ -23,17 +24,19 @@ export const SubPageWrapper = ({
   return (
     <div
       style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 24,
```

Comment:
this is the container for the workflow step sidebar and email/in-app editors... the items should be spaced equally